app/scripts/modules/kubernetes/serverGroup/configure/wizard/deployment.html

```diff
@@ -0,0 +1,60 @@
+<form class="container-fluid form-horizontal">
+  <ng-form name="body">
+    <div class="form-group">
+      <div class="col-md-3 sm-label-right">
+        Deployment
+        <help-field key="kubernetes.serverGroup.deployment.enabled"></help-field>
+      </div>
+      <div class="col-md-2">
+        <input type="checkbox" name="details" ng-model="command.deployment.enabled"/>
+      </div>
+    </div>
+    <div ng-show="command.deployment.enabled">
+    </br>
```

Comment:
I don't see an opening `<br> tag, so guessing this was supposed to be a `\<br>` or `\<br/>`. Also looks like it's not indented properly.